_pages/cbio-who.md

```diff
@@ -73,6 +73,12 @@ sidebar:
   part of the large, inter-institute
   [HYGIEIA](https://www.saintluc.be/fr/hygieia) project.
 
+  **Enes Sefa Ayar** (Github:
+  [EnesSefaAyar]((https://github.com/EnesSefaAyar)) joined the lab in
+  November 2023 as a PhD student working on quantitative analysis of
+  single-cell proteomics. He did a master's in computational sciences
+  and engineering at the Koç University. He is funded by a FNRS fellowship.
```

Comment:
Could you change the last part with 

He is funded by a FNRS 'Projet de Recherche' (PDR) awarded to L. Gatto.